tests/tests_system/test_os.py

```diff
@@ -14,7 +14,7 @@
 class TestOs(unittest.TestCase):
     # current_architecture
     def test_current_architecture(self) -> None:
-        self.assertTrue(current_architecture() in ["x64", "arm64"])
+        self.assertTrue(current_architecture() in ["x64", "arm64", "ppc64le"])
```

Comment:
@Sunidhi-Gaonkar1 you need to add one more test case:
```


src/system/os.py#L19

Added line #L19 was not covered by tests


```

Reply:
I have made the changes, Please review. Thank you.